src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -484,6 +485,17 @@ public void cannotCheckOutWhenItemDeclaredLost() {
       hasItemBarcodeParameter(declaredLostItem))));
   }
 
+  @Test
+  public void cannotCheckOutWhenItemAgedToLost() {
+    val agedToLostItem = itemsFixture.basedUponNod(ItemBuilder::agedToLost);
+
+    final Response response = checkOutFixture.attemptCheckOutByBarcode(
+      agedToLostItem, usersFixture.steve());
+
+    assertThat(response.getJson(),
+      hasErrorWith(hasMessageContaining("has the item status Aged to lost")));
+  }
+
```

Comment:
Could this be converted to a unit test for the `ItemStatusValidator`?

Reply:
We probably then need a API test, that demonstrates that check-out rejected for an item that falls under this validation rule, e.g. `checkOutRejectedForAnDissalowedItemStatus`. Because the unit test only proves that the validator works correctly but we have to verify that it works as expected in integration.

Reply:
I completely agree (with your reasoning) that we need one API test to demonstrate that the validator is used. 

To use the form that you've used for other tests, I suggest something like `shouldRejectCheckOutOfItemInDisallowedStatus`

Reply:
Please can the existing and new tests be consolidated into that single API test

Reply:
Done